packages/gasket-cli/src/scaffold/actions/global-prompts.js

```diff
@@ -11,7 +11,7 @@ const { addPluginsToContext } = require('../utils');
  */
 async function chooseAppDescription(context) {
   if (!('appDescription' in context)) {
-    const { appDescription } = await inquirer.prompt([
+    const { appDescription } = context.description || await inquirer.prompt([
```

Comment:
Hmm, are you intended to normalize `context.description` with `context.appDescription` here? If `context.description` is set to a string, as in your tests, it's going to break down trying to destructure that here as `{ appDescription }`.

I don't think we need to introduced a new `context.description`. Instead, just tune up your tests to to use the existing `context.appDescription`.